test/e2e/pivoting.go

```diff
@@ -83,6 +83,15 @@ func pivoting(ctx context.Context, inputGetter func() PivotingInput) {
 		fetchContainerLogs(&ironicContainers, input.ArtifactFolder, input.E2EConfig.GetVariable("CONTAINER_RUNTIME"))
 	}
 
+	By("Fetch target cluster kubeconfig and save for target log collection")
+	kubeconfigFile := "/tmp/kubeconfig-test1.yaml"
+	kubeconfig := input.TargetCluster.GetKubeconfigPath()
```

Comment:
Here we can use:
`os.Setenv("KUBE_CONFIG", "kubeconfig")`
and remove  from the script
`KUBE_CONFIG="$(sudo find /tmp/ -type f -name "kubeconfig*")"`

Reply:
I agree that this is an option. But the log collection from project-infra will not work as it couldn't find the kubeconfig locally.